src/sentry/replays/usecases/query/__init__.py

```diff
@@ -265,7 +294,55 @@ def query_using_optimized_search(
         referrer="replays.query.browse_query",
     )["data"]
 
-    return _make_ordered(replay_ids, results), has_more
+    return QueryResponse(
+        response=_make_ordered(replay_ids, results),
+        has_more=has_more,
+        source=source,
+    )
+
+
+def make_materialized_view_search_query(
+    search_filters: Sequence[SearchFilter | str | ParenExpression],
+    sort: str | None,
+    project_ids: list[int],
+    period_start: datetime,
+    period_stop: datetime,
+) -> Query:
+    orderby = handle_ordering(mv.sort_config, sort or "-" + DEFAULT_SORT_FIELD)
+
+    having: list[Condition] = handle_search_filters(mv.search_config, search_filters)
+    having.append(Condition(Function("minMerge", parameters=[Column("min_segment_id")]), Op.EQ, 0))
+
+    return Query(
+        match=Entity("replays_aggregated"),
+        select=[Column("replay_id")],
+        where=[
+            Condition(Column("project_id"), Op.IN, project_ids),
+            Condition(Column("to_hour_timestamp"), Op.LT, period_stop),
+            Condition(Column("to_hour_timestamp"), Op.GTE, period_start),
+        ],
+        having=having,
+        orderby=orderby,
+        groupby=[Column("replay_id")],
+    )
+
+
+def make_materialized_view_query(
```

Comment:
don't notice any references to this fx, what was it intended for? @cmanallen

Reply:
No clue.  Drop it if its not useful to you.